cmake/FindCoroutines.cmake

```diff
@@ -108,12 +108,12 @@ cmake_push_check_state()
 
 set(CMAKE_REQUIRED_QUIET ${Coroutines_FIND_QUIETLY})
 
-# All of our tests required C++20 or later
+# All of our tests required C++17 or later
 if(("x${CMAKE_CXX_COMPILER_ID}" MATCHES "x.*Clang" AND "x${CMAKE_CXX_SIMULATE_ID}" STREQUAL "xMSVC") OR "x${CMAKE_CXX_COMPILER_ID}" STREQUAL "xMSVC")
-    set(_CXX_COROUTINES_STD20 "/std:latest")
+    set(_CXX_COROUTINES_STD17 "/std:c++17")
```

Comment:
How does this flag end up being overridden by the -DCMAKE_CXX_STANDARD flag?
Is that some CMake magic behaviour?

Reply:
Ah, never mind. I see it's just checking whether it _can_ use coroutines at all.
This flag isn't actually being passed to libunifex sources.